python/lsst/ts/observatory/control/maintel/mtcalsys.py

```diff
@@ -200,6 +206,27 @@ async def setup_calsys(self, sequence_name: str) -> None:
             await self.linearstage_projector_select.cmd_moveAbsolute.set_start(
                 distance=self.ls_select_led_location, timeout=self.long_timeout
             )
+        elif (calibration_type == CalibrationType.CBPCalibration) or (
+            calibration_type == CalibrationType.CBP
```

Comment:
As I mentioned above, I thinks CalibrationType should remain the "light input" and we should create a new variable for the CBP vs. FlatField